EpiAware/README.md

```diff
@@ -81,3 +89,13 @@ H-.->I
 ## Pluto scripts
 
 We use [`Pluto.jl`](https://plutojl.org/) scripts as part of our documentation and testing. The scripts are located in `docs/src/examples` and can be run using the `Pluto.jl` package. We recommend using the version of `Pluto` that is pinned in the `Project.toml` file defining the documentation environment. An entry point to running or developing this documentation is the `docs/pluto-scripts.sh` bash shell script. Run this from the root directory of this repository.
```

Comment:
Will this activate the docs environment for people?

Reply:
This also now suggests a clone followed by running a bash script. Can pluto do anything smart about pointing at a URL?

Reply:
> Will this activate the docs environment for people?
Yes, the bash script does that.

Reply:
> This also now suggests a clone followed by running a bash script. Can pluto do anything smart about pointing at a URL?

Good spot. There is a tension here because on one hand we're using Pluto as a document generating platform (inspired by how [`TuringGLM.jl`](https://github.com/TuringLang/TuringGLM.jl) does docs), in which case then this is fine but otoh suggesting a script.

Not sure what the right balance is here.

Reply:
In other packages generally you run tutorials locally and the onus is on the user to choose how to do that (e.g. C&P into REPL, use a Jupyter nb, use a Pluto nb etc etc), and the dev of the documentation is not added.

On the other extreme, we can be fancy and use something like binder for Pluto. There might be some pain there, because Pluto as default is self-contained (e.g. installs its own package version as part of a nb specific env) whereas here it isn't. For example, we use the Pluto nbs for dev as well as docs and therefore we prefer a pattern of using `Pkg.develop` on `EpiAware` rather than `Pkg.add` (which I think is easier for doing binder on but I might be mistaken).

Reply:
Yeah this makes this makes sense. My worry is essentially we are expecting a lot of people who are new to Julia to look at this and we want to make it as easy as possible.. 

For now let's leave as is and come back to it if we think of anything that makes working with it easier.